### PR TITLE
Added the current sponsors

### PR DESCRIPTION
**Pull request by DieuwerH**
Added the current sponsors: ProDrive and Spam

Also fixed overflow of sponsor elements into next section

### CODE REVIEW

File: src/lancie-sponsors/lancie-sponsors.html [hunk 1]
```diff
@@ -48,19 +53,12 @@ <h3>Presented by</h3>
...
-          <a href="https://ogd.nl/" target="_blank" rel="noopener noreferrer"><img src="../../images-optimized/logos/ogd.png"></a>
-        </div>
-      <div class="layout vertical flex-4">
-        <h3>Sponsors</h3>
-        <div class="layout horizontal wrap regular-sponsors">
-          <a href="http://www.coolermaster.com/" target="_blank" rel="noopener noreferrer"><img src="../../images-optimized/logos/Cooler_Master_Logo.png"></a>
-          <a href="http://www.spam-energydrink.com/" target="_blank" rel="noopener noreferrer"><img src="../../images-optimized/logos/spam-logo.jpg"></a>
-          <a href="http://www.jigsaw.nl/" target="_blank" rel="noopener noreferrer"><img id="jigsawLogo" src="../../images-optimized/logos/jigsaw_cleaner_logo.png"></a>
-          <a href="https://www.transip.nl/" target="_blank" rel="noopener noreferrer"><img src="../../images-optimized/logos/transip_logo.png"></a>
+        <div class="layout vertical flex-4">
```
Comment (Fastjur) [inline src/lancie-sponsors/lancie-sponsors.html]: Is this supposed to be twice the same thing inside each other?

File: src/lancie-home-page/lancie-home-page.html [hunk 2]
```diff
@@ -99,9 +97,11 @@
       </div>
     </lancie-section>
 
-    <!--<lancie-section type="primary" header="Partners &amp; Sponsors">
+    <lancie-info-blocks></lancie-info-blocks>
```
Comment (FrancisBehnen) [inline src/lancie-home-page/lancie-home-page.html]: Shouldn't the info blocks be above the updates section?

Comment (martijnjanssen) [reply src/lancie-home-page/lancie-home-page.html]: Haha yeah, they should... 😂

**Comment by FrancisBehnen** [comment]
ROFL, now the same `div` closing is missing again

File: src/lancie-home-page/lancie-home-page.html (hunk 2 above)
Comment (DieuwerH) [reply src/lancie-home-page/lancie-home-page.html]: I put it below the updates section so the background pattern (blue white blue) would stay the same. The sponsor section needs a white background for sponsor logo's with transparent backgrounds (blue doesn't look good). If the info blocks is put above the updates section you would have two white backgrounds after each other

File: src/lancie-sponsors/lancie-sponsors.html [hunk 3]
```diff
@@ -27,6 +27,11 @@
         margin: auto 6px;
       }
 
+      .regular-sponsors a {
+        height: 100%;
+        margin: auto 6px;
+      }
+
       #jigsawLogo {
```
Comment (martijnjanssen) [inline src/lancie-sponsors/lancie-sponsors.html]: Please also remove css that is no longer needed.

File: src/lancie-home-page/lancie-home-page.html [hunk 4]
```diff
@@ -86,23 +86,23 @@
...
+
+    <lancie-section type="primary" header="Partners &amp; Sponsors">
+      <lancie-sponsors></lancie-sponsors>
+    </lancie-section>
+
     <lancie-section type="primary" header="Updates">
       <div>
         <p>
-          Don't worry, we hate spam too. We will not share your details with anyone.<br/>
+          Don't worry, we hate junkmail too. We will not share your details with anyone.<br/>
```
Comment (martijnjanssen) [inline src/lancie-home-page/lancie-home-page.html]: Nice 👍 .

**Comment by Fastjur** [comment]
I set the colors to be alternating again.  
I am not sure what to think of it. Any thoughts?


image

**Comment by skamoen** [comment]
I assume the sign up thingy will go away soon so it'll be back to normal afterwards right?

**Comment by Fastjur** [comment]
Yeah we could also remove the signup thingy perhaps

**Pull request merged**